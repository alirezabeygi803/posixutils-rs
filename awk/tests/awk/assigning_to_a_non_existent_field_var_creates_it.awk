$1 == 1 {
    print NF
    print $0
    print $4
    $4 = 1
    print $4
    print NF
    print $0

    print $10
    $10 = "test"
    print $10
    print NF
    print $0
}