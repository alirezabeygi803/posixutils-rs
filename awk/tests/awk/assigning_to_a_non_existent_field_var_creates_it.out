4
1 Jane janitor 30
30
1
4
1 Jane janitor 1

test
10
1 Jane janitor 1      test
